src/Command/ExplainAdminCommand.php

```diff
@@ -33,15 +31,9 @@ final class ExplainAdminCommand extends Command
      */
     private $pool;
 
-    /**
-     * @var MetadataFactoryInterface
-     */
-    private $validator;
-
-    public function __construct(Pool $pool, MetadataFactoryInterface $validator)
+    public function __construct(Pool $pool)
```

Comment:
Shall we make this class `@internal` to remove it from the BC promise?

Reply:
Done for all the command and twig extensions.

Reply:
> to me this class is not meant to be extended

the class is final already? So not sure I see the benefit here

Reply:
I understand it for the constructor signature